xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/pvr/EPG.h

```diff
@@ -434,19 +434,32 @@ class PVREPGTag : public CStructHdl<PVREPGTag, EPG_TAG>
 
   void SetData(const EPG_TAG* tag)
   {
-    m_title = tag->strTitle;
-    m_plotOutline = tag->strPlotOutline;
-    m_plot = tag->strPlot;
-    m_originalTitle = tag->strOriginalTitle;
-    m_cast = tag->strCast;
-    m_director = tag->strDirector;
-    m_writer = tag->strWriter;
-    m_IMDBNumber = tag->strIMDBNumber;
-    m_iconPath = tag->strIconPath;
-    m_genreDescription = tag->strGenreDescription;
-    m_episodeName = tag->strEpisodeName;
-    m_seriesLink = tag->strSeriesLink;
-    m_firstAired = tag->strFirstAired;
+    if (tag->strTitle != nullptr)
+      m_title = tag->strTitle;
```

Comment:
Maybe it could also:
```cpp
    m_title = tag->strTitle == nullptr ? "" : tag->strTitle;
    m_plotOutline = tag->strPlotOutline == nullptr ? "" : tag->strPlotOutline;
    m_plot = tag->strPlot == nullptr ? "" : tag->strPlot;
    m_originalTitle = tag->strOriginalTitle == nullptr ? "" : tag->strOriginalTitle;
    m_cast = tag->strCast == nullptr ? "" : tag->strCast;
    m_director = tag->strDirector == nullptr ? "" : tag->strDirector;
    m_writer = tag->strWriter == nullptr ? "" : tag->strWriter;
    m_IMDBNumber = tag->strIMDBNumber == nullptr ? "" : tag->strIMDBNumber;
    m_iconPath = tag->strIconPath == nullptr ? "" : tag->strIconPath;
    m_genreDescription = tag->strGenreDescription == nullptr ? "" : tag->strGenreDescription;
    m_episodeName = tag->strEpisodeName == nullptr ? "" : tag->strEpisodeName;
    m_seriesLink = tag->strSeriesLink == nullptr ? "" : tag->strSeriesLink;
    m_firstAired = tag->strFirstAired == nullptr ? "" : tag->strFirstAired;
```
To confirm and have in all cases the values set (also if empty and the maybe if `SetData` used in future for other things)?

Reply:
Sure thing, I can change the PR later today to use this method instead.

Reply:
+1, because I feel this is more readable code